opflexagent/utils/ep_managers/endpoint_file_manager.py

```diff
@@ -342,13 +342,38 @@ def _mapping_to_file(self, port, mapping, fixed_ips):
                     self._map_dhcp_info(fixed_ips, mapping,
                                         mapping_dict)
         ips_aap = []
-        for aap in mapping.get('allowed_address_pairs', []):
+
+        def filter_cidr_aaps(aaps):
+            # Remove any CIDRs in the AAP list
+            cidr_aaps = [aaps.pop(index)
+                         for index in range(len(aaps))[::-1]
+                         if netaddr.IPNetwork(
+                             aaps[index]['ip_address']).prefixlen != 32]
```

Comment:
will need to also consider ipv6 case per Bob's comment on the corresponding GBP patch.

Reply:
Good point -- will fix.